src/main/java/seedu/address/ui/DoctorCard.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.ui;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Doctor;
+
+/**
+ * An UI component that displays information of a {@code Doctor}.
+ */
+public class DoctorCard extends UiPart<Region> {
+
+    private static final String FXML = "DoctorListCard.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Doctor doctor;
+
+    @FXML
+    private HBox cardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label nric;
+    @FXML
+    private Label type;
+
+    public DoctorCard(Doctor doctor, int displayedIndex) {
+        super(FXML);
+        this.doctor = doctor;
+        id.setText(displayedIndex + ". ");
+        name.setText(doctor.getName().fullName);
+        phone.setText("Phone Number: " + doctor.getPhone().value);
```

Comment:
Do we want to be consistent and provide label for all the field in the UI? 
Name: aaa
Phone: 123
etc
because right now, if I am not wrong, only the phone have the label

Reply:
We alr displaying all the field tho?

Reply:
I think Simnee meant `name.setText("Name: " + doctor.getName().fullName);`

Reply:
I'm not familiar with how the UI elements are tying together yet, but I don't think there's a need to do what you're suggesting? I think other UI elements are already taking care of it.

This is how it looks like right now:
![image](https://user-images.githubusercontent.com/43085796/66703852-756d3e80-ed49-11e9-9f0c-7328149339c1.png)

If you're all ok with it I'll merge

Reply:
Ok sure I didnt notice that it will become a tag beside the name. Thanks for the pic

Reply:
Thanks! Ill merge ba, since you're the main person concerned with this and @Yen-Peng is just clarifying